extensions/wikia/CategoryPage3/templates/trending-pages.php

```diff
@@ -0,0 +1,18 @@
+<div class="category-page__trending-pages-header"><?= wfMessage( 'category-page3-trending-pages' ) ?></div>
```

Comment:
The default output format of `wfMessage` is `parse`. Since this message contains no wikitext, to save using the parser, I'd explicitly use `->escaped()` here.